java/test/src/main/java/org/ray/api/test/RaySerializerTest.java

```diff
@@ -0,0 +1,34 @@
+package org.ray.api.test;
+
+import com.google.gson.internal.LinkedTreeMap;
+import java.util.Map;
+import org.ray.api.RayPyActor;
+import org.ray.api.id.UniqueId;
+import org.ray.runtime.RayPyActorImpl;
+import org.ray.runtime.util.Serializer;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RaySerializerTest {
+
+  @Test
+  public void testLinkedTreeMapSer() {
```

Comment:
I think it's not necessary to add this test in Ray's code base.

Reply:
Make sense. This test should be in FST repo.